plugins/services/src/js/pages/task-details/TaskIpAddressesRow.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Application from "#PLUGINS/services/src/js/structs/Application";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import DCOSStore from "#SRC/js/stores/DCOSStore";
+import Pod from "#PLUGINS/services/src/js/structs/Pod";
+
+export default class TaskIpAddressesRow extends React.Component {
+  getIPAddressesForTask(service, taskId) {
```

Comment:
Also possible, just out of curiosity, why did you not use a function instead of a method?

Reply:
It wasn't difficult to test it with snapshots and I decided keeping it in the class rather then extracting it.

Reply:
okay 👍